apps/angular/src/app/features/anime/pagination/pagination.component.ts

```diff
@@ -0,0 +1,46 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { PageEvent } from '@angular/material/paginator';
+
+/** Pagination options. */
+@Component({
+  selector: 'camp-pagination',
+  templateUrl: './pagination.component.html',
+  styleUrls: ['./pagination.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+
+/** Init pagination component. */
+export class PaginationComponent {
+
+  /** Change event component.
+   * @param event Event page change.
+   */
+  @Output() public pageChange = new EventEmitter<PageEvent>();
+
+  /** Total items of pagination. */
+  @Input() public totalItems: number | null;
```

Comment:
Same goes for other decorators.
```suggestion
  @Input()
  public totalItems: number | null;
```